package.json

```diff
@@ -67,6 +67,6 @@
     "beachball": "^2.45.0",
     "concurrently": "^9.0.0",
     "patch-package": "^8.0.0",
-    "playwright": "1.54.1"
+    "playwright": "^1.57.0"
```

Comment:
There seems to be a bunch of unrelated changes outside of the react workspace changing dependencies, etc. Can you revert these unnecessary changes?